### PR TITLE
Remove  `get_project` function

### PR DESCRIPTION
**Pull request by AleksanderWWW**
## Before submitting checklist

- [x] Did you **update the CHANGELOG**? (not for test updates, internal changes/refactors or CI/CD setup)
- [x] Did you **asked the docs owner** to review all the updated user-facing interfaces?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ### Changes
 - Moved modules from `neptune.new` to `neptune` with compatibility imports and marked `neptune.new` as deprecated ([#1213](https://github.com/neptune-ai/neptune-client/pull/1213))
 - Removed `neptune.*` legacy modules ([#1206](https://github.com/neptune-ai/neptune-client/pull/1206))
+- Deprecated `get_project` function ([#1214](https://github.com/neptune-ai/neptune-client/pull/1214))
```
Comment (normandy7) [inline CHANGELOG.md:6]: Correct me if I'm wrong, but it's deprecated already and actually removed with 1.0, no?

Comment (AleksanderWWW) [reply CHANGELOG.md]: Yep

File: src/neptune/new/sync/__init__.py [hunk 2]
```diff
@@ -36,7 +36,6 @@
     "SyncRunner",
     "get_metadata_container",
     "get_offline_dirs",
-    "get_project",
```
Comment (Raalsky) [inline src/neptune/new/sync/__init__.py:39]: That's too far. It's different function

File: src/neptune/new/sync/utils.py [hunk 3]
```diff
@@ -16,7 +16,6 @@
 
 __all__ = [
     "get_metadata_container",
-    "get_project",
```
Comment (Raalsky) [inline src/neptune/new/sync/utils.py:19]: That's too far. It's different function #2

**Review by Raalsky — CHANGES_REQUESTED** [review]

**Review by Raalsky — APPROVED** [review]

**Pull request merged**